tests/negative/partitions/zeroes.go

```diff
@@ -65,8 +65,8 @@ func Partition9DoesNotStartCorrectly() types.Test {
 	name := "partition.match.failstostart"
 	in := types.GetBaseDisk()
 	//insert a gap before 9
-	tmp := in[0].Partitions[9-2-1]
-	in[0].Partitions[9-2-1] = &types.Partition{
+	tmp := in[0].Partitions[9-6-1]
```

Comment:
Is it still worth keeping the `9-<offset>-1` format?

Reply:
I want to rework a lot of these later, so I was going to hold off until then.